src/scribe_data/cli/utils.py

```diff
@@ -0,0 +1,60 @@
+import json
+from pathlib import Path
+from typing import Dict, List, Union
+
+# Load language metadata from JSON file
+METADATA_FILE = Path(__file__).parent.parent / 'resources' / 'language_meta_data.json'
+
+def load_language_metadata() -> Dict:
+    with METADATA_FILE.open('r', encoding='utf-8') as file:
+        return json.load(file)
+
+LANGUAGE_METADATA = load_language_metadata()
+LANGUAGE_MAP = {lang['language'].lower(): lang for lang in LANGUAGE_METADATA['languages']}
+
+def print_formatted_data(data: Union[Dict, List], word_type: str) -> None:
```

Comment:
Let's keep this copy of `print_formatted_data` as I think it makes sense to have it in utils :)